qiita_db/util.py

```diff
@@ -422,17 +420,17 @@ def compute_checksum(path):
     else:
         filepaths.append(path)
 
+    buffersize = 65536
+    crcvalue = 0
     for fp in filepaths:
-        with open(fp, 'rb', newline=None) as f:
-            # Go line by line so we don't need to load the entire file
-            for line in f:
-                if crc is None:
-                    crc = crc32(line)
-                else:
-                    crc = crc32(line, crc)
-    # We need the & 0xffffffff in order to get the same numeric value across
+        with open(fp, 'rb') as f:
```

Comment:
Just to confirm, removing 'newline=None' from the open call was intended?

Reply:
Yup ... it's automatic.